ram/wram0.asm

```diff
@@ -1392,3 +1392,12 @@ wDaysSince:: db
 
 ; Temporary backup for options
 wOptionsBuffer:: db
+
+	ds 1 ; unused
+
+; Contains a copy of the rom checksum, read from the header. Used as
+; protection against people trying to load a save state for a save in
+; a different rom version.
+wRomChecksum:: dw
+
+assert @ == WRAM0_End
```

Comment:
`wRomChecksum` could have its own section, `org`ed in the linkerscript.

Reply:
put it there simply because there were 3 bytes of slack from options already having an org. Would a separate section be preferable?

Reply:
Yes please, instead of an `assert`. You can redundantly do `ROMX[$dffe]` too, same as how the header sections are fixed.